tests.lisp

```diff
@@ -0,0 +1,42 @@
+
+(defpackage #:external-symbol-not-found.test
+  (:documentation "Tests for external-symbol-not-found.")
+  (:use #:cl)
+  (:import-from
+   #:external-symbol-not-found
+   #:external-symbol-not-found-p)
+  (:import-from
+   #:parachute
+   #:define-test
+   #:true))
+
+(in-package #:external-symbol-not-found.test)
+
+(defmacro with-temp-package ((name) &body body)
+  `(unwind-protect
+	(progn
+	  (make-package ',name :use nil)
+	  ,@body)
+     (delete-package ',name)))
+
+(define-test external-symbol-not-found
+  #+(not (or sbcl ccl ecl abcl)) (not-supported-error))
```

Comment:
I actually think it should be possible to support more implementations - but that's topic a for another issue.

Reply:
Yes, I was going to try to support more after adding the CI.